core/server/common/src/main/java/alluxio/cli/validation/ValidationTask.java

```diff
@@ -19,5 +21,5 @@
    * Runs the validation task.
    * @return whether the validation succeeds
    */
-  boolean validate() throws InterruptedException;
+  boolean validate(String[] args, Map<String, String> optionMap) throws InterruptedException;
```

Comment:
Is `args` used in any validation tasks?

Reply:
Not now. Maybe I should remove it. In case we need it in the future, I can add it back.